numba_rvsdg/core/datastructures/scfg.py

```diff
@@ -845,14 +942,46 @@ def to_dict(self):
         graph_dict: Dict[Dict[...]]
             A dictionary representing the SCFG.
         """
-        scfg_graph = self.graph
-        graph_dict = {}
-        for key, value in scfg_graph.items():
-            curr_dict = {}
-            curr_dict["jt"] = [i for i in value._jump_targets]
-            if value.backedges:
-                curr_dict["be"] = [i for i in value.backedges]
-            graph_dict[key] = curr_dict
+        blocks = {}
+        edges = {}
+        backedges = {}
+
+        def reverse_lookup(value):
```

Comment:
this needs a type signature

Reply:
The `value`s are class constructors, not sure what type that is.